.eslintrc.json

```diff
@@ -10,6 +10,10 @@
         "node": true,
         "es6": true
     },
+    "extends": [
+        "plugin:@typescript-eslint/recommended",
```

Comment:
Should this also include eslint's own recommended ones?

Reply:
Ah, I do tend to forget that one... thanks.

Reply:
Thanks; there's a bunch of plain eslint rules leftover at the bottom, I'll go check to see if any can be removed.